src/main/java/chess/domain/position/Position.java

```diff
@@ -7,6 +7,7 @@
 import java.util.Objects;
 
 public class Position {
+
     private static final Map<String, Position> POSITIONS = new HashMap<>();
     private final static int POSITION_ROW_INDEX = 1;
```

Comment:
```suggestion
    private static final int POSITION_ROW_INDEX = 1;
```
다른 static도 순서가 맞는지 점검해봅시다.